controllers/api.js

```diff
@@ -19,6 +20,20 @@ const api = {
    	  response.success = false;
      }
      res.send(response);
+  },
+
+  loggedIn : function (req, res, next) {
+     if (req.user) {
+        // if loggedIn, allow normal api access
+        next()
+     } else {
+        console.log("No user associated with session; redirecting")
+        res.redirect('/api/login/')
+     }
+  },
+
+   serveHome : function (req, res, next) {
```

Comment:
I feel like the api shouldn't be ever serving pages, it shouldn't assume the user wants html.